wallet/android/app/src/main/res/xml/file_paths.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths>
+    <external-path
```

Comment:
we need to modify this. we cannot export all the app data . Kindly check which one we actually need to export.